CHANGES.txt

```diff
@@ -5,7 +5,27 @@ CHANGELOG
 3.3.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Cornice now only accepts Marshmallow schemas instead of instances
+- Marshmallow schemas get request object added to their context
+  during validation
```

Comment:
You can mention the pull-request number here

Reply:
Fixed